README.md

```diff
@@ -1,32 +1,25 @@
-# Show Whitespace in Brackets
+# Show Whitespace
 
-[Brackets](http://brackets.io/) extension to visualize whitespace (spaces and tabs) like [Sublime Text](http://www.sublimetext.com/) does. Adds an entry called "Show Whitespace" to the View menu, the shortcut is Command-Shift-W (or Ctrl-Shift-W on Windows). The state is remembered next time you start Brackets.
+> A [Brackets](http://brackets.io/) extension to visualize whitespace (both spaces and tabs) in the same style as [Sublime Text](http://www.sublimetext.com/).
 
-This extension overwrites `Line.getHTML` (part of [CodeMirror](http://codemirror.net/)) and modifies its output. Spaces in the indentation are wrapped as `<span class="cm-space dk-leading"> </span>`, normal spaces as `<span class="cm-space dk-normal"> </span>`. CodeMirror itself already handles tabs similarly, this extension turns those into `<span class="cm-tab dk-leading"> </span>` if they are part of the whitespace starting a line and `<span class="cm-tab dk-normal"> </span>` otherwise.
+## Usage
 
-The styles are defined in `main.less` which is compiled to CSS and loaded into Brackets when loading the extension.
+Activate using `View > Show Whitespace` or by using the shortcut `Ctrl-Shift-W` (`Command-Shift-W` on Mac OS X). The extension state is remembered across Brackets launches. Whitespace in all editor windows are visualized, including inline editors.
 
-Whitespace in inline editors is visualized as well.
+This extension uses [CodeMirror](http://codemirror.net/) overlays to construct and render whitespace for both spaces and tabs. Each individual blank character are wrapped in a `<span></span>` with an appropriate denoting class. The following are all the classes this extension uses and their purpose. Do note that tabs follow this same structure, with the word `space` replaced with `tab`.
```

Comment:
Just realized I might want a header here to indicate this is technical details. How about "Technical Details" or "How It Works"?